.env.trailbase

```
@@ -0,0 +1,52 @@
+PPN_RATE=11.0
+PPH23_CONSTRUCTION_RATE=2.0
+BPJS_KESEHATAN_EMPLOYEE=1.0
+BPJS_KETENAGAKERJAAN_EMPLOYEE=2.0
+
+# File Management
+MAX_FILE_SIZE=104857600
+ALLOWED_MIME_TYPES=image/jpeg,image/png,image/gif,application/pdf,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
+
+# Security
+SECRET_KEY=dev-secret-key-change-in-production
+JWT_SECRET_KEY=dev-jwt-secret-key-change-in-production
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JWT_SECRET_KEY key should go before the SECRET_KEY key

(UnorderedKey)